layers/+lang/python/packages.el

```diff
@@ -95,6 +96,19 @@
       (add-to-list 'spacemacs-jump-handlers-python-mode
                    '(anaconda-mode-find-definitions :async t)))))
 
+(defun python/init-code-cells ()
+  (use-package code-cells
+    :if (not (configuration-layer/layer-used-p 'ipython-notebook))
+    :defer t
+    :init
+    (progn
+      (add-hook 'python-mode-hook 'code-cells-mode)
+      (spacemacs/set-leader-keys-for-major-mode 'python-mode
```

Comment:
Could `spacemacs/set-leader-keys-for-minor-mode 'code-cells-mode` be more appropriate there?

Reply:
good idea indeed!